ui/pages/home/home.component.js

```diff
@@ -653,23 +749,51 @@ export default class Home extends PureComponent {
           exact
         />
         <div className="home__container">
-          {showWhatsNew ? <WhatsNewPopup onClose={hideWhatsNewPopup} /> : null}
-          {!showWhatsNew && showRecoveryPhraseReminder ? (
-            <RecoveryPhraseReminder
-              hasBackedUp={seedPhraseBackedUp}
-              onConfirm={this.onRecoveryPhraseReminderClose}
-            />
-          ) : null}
-          {showTermsOfUse ? (
-            <TermsOfUsePopup onAccept={this.onAcceptTermsOfUse} />
-          ) : null}
-          {isPopup && !connectedStatusPopoverHasBeenShown
-            ? this.renderPopover()
-            : null}
+          {
+            ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
+            showWhatsNew ? <WhatsNewPopup onClose={hideWhatsNewPopup} /> : null
+            ///: END:ONLY_INCLUDE_IN
+          }
+          {
+            ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
+            !showWhatsNew && showRecoveryPhraseReminder ? (
+              <RecoveryPhraseReminder
+                hasBackedUp={seedPhraseBackedUp}
+                onConfirm={this.onRecoveryPhraseReminderClose}
+              />
+            ) : null
+            ///: END:ONLY_INCLUDE_IN
+          }
+          {
+            ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
+            showTermsOfUse ? (
+              <TermsOfUsePopup onAccept={this.onAcceptTermsOfUse} />
+            ) : null
+            ///: END:ONLY_INCLUDE_IN
+          }
+          {
+            ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
+            isPopup && !connectedStatusPopoverHasBeenShown
+              ? this.renderPopover()
+              : null
+            ///: END:ONLY_INCLUDE_IN
+          }
```

Comment:
I'm not a master code fencer, but do the code fencing comments work in a react tree if you put them in new code blocks? aka instead of what you have here 

```jsx
<div>
  { ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask) }
  { ...some other code}
  { ...some more code }
  { ///: END:ONLY_INCLUDE_IN }
</div>
```

@FrederikBolding have you experimented with this by chance?

Reply:
With the last curly } in the line bellow, it should work:
<div>
  {
   ///: BEGIN:ONLY_INCLUDE_IN(build-main,build-beta,build-flask)
  }
  { ...some other code}
  { ...some more code }
  {
  ///: END:ONLY_INCLUDE_IN
  }
</div>

Reply:
Might make this just a bit easier to read but I am unsure if it will make sense. Open to thoughts

Reply:
Agree, I also prefer it like this, better readability 👍🏼